package/yast2-update.spec

```diff
@@ -30,6 +30,7 @@ BuildRequires:  yast2-devtools >= 3.1.15
 BuildRequires:  yast2-ruby-bindings >= 1.0.0
 BuildRequires:  yast2 >= 3.1.126
 BuildRequires:  yast2-packager
+BuildRequires:  yast2-storage
```

Comment:
Um, I do not know how much is the *yast2-storage* needed for running tests, but maybe we could mock it completely to avoid long build dependencies.

See [this example](https://github.com/yast/yast-installation/blob/master/test/test_helper.rb#L17), here *yast2-installation* mocks some AutoYaST modules to avoid the build dependency on AutoYaST.